### ISSUE TITLE
Small Grammar issue outputs an error to the log

### ISSUE THREAD

**Issue by TheComputerizer** [TheUnderTaker11/More-Avaritia#40]
I notice this error in my log when starting up my pack
```
Parsing error loading recipe moreavaritia:items/dirt
com.google.gson.JsonSyntaxException: Unknown item 'mineraft:stone
```
which seems to indicate that minecraft is misspelled as mineraft in that recipe causing it to not work

**Comment by loliconRoot** [comment]
I had the same problem with More Avaritia MC 1.12.2 v1.4. The source code on Github doesn't seem to be up to date anymore, so I can't even fix the problem by modifying it.

**Comment by TheUnderTaker11** [comment]
> I had the same problem with More Avaritia MC 1.12.2 v1.4. The source code on Github doesn't seem to be up to date anymore, so I can't even fix the problem by modifying it.

Wow never realized that, sorry. Just pushed updated code.

**Comment by TheUnderTaker11** [comment]
@loliconRoot

**Comment by TheUnderTaker11** [comment]
Just looked and it's a super expensive recipe for dirt, so not that big of a deal lol. Not any special dirt, just minecraft:dirt.

Probably put it there when I was testing the JSON recipe format to transition recipes to.

Either way, fixed it in the code for if I ever release another file any time soon.

**Comment by TheUnderTaker11** [comment]
To answer the when question, I have just released v1.5 which fixes this bug and #34 , code has just been updated to github

**Comment by TheComputerizer** [comment]
Wow I was not expecting this to still be fixed, thanks for that.
